client/my-sites/stats/activity-log-day/index.jsx

```diff
@@ -76,6 +76,18 @@ class ActivityLogDay extends Component {
 		dayExpanded: false,
 	};
 
+	shouldComponentUpdate( nextProps ) {
+		return (
+			nextProps.logs.length !== this.props.logs.length ||
```

Comment:
this is going to prevent updates when the logs change

Reply:
I tested live update with this and it works. When the number of logs is different, the day updates. In any case I'm going to close it since you said on Monday that you'll be rewriting some areas of AL that will end up in a solution for this.